cmd/data-crawler.go

```diff
@@ -490,12 +491,16 @@ func (f *folderScanner) scanQueuedLevels(ctx context.Context, folders []cachedFo
 
 			err = objAPI.HealObjects(ctx, bucket, prefix, madmin.HealOpts{Recursive: true, Remove: healDeleteDangling},
 				func(bucket, object, versionID string) error {
+					// Wait for each heal as per crawler frequency.
+					sleepDuration(time.Since(t), f.dataUsageCrawlMult)
```

Comment:
This will wait progressively longer for each object since `t` isn't reset.

I guess you can defer a reset.

Reply:
Fixed @klauspost